src/math/ceil.js

```diff
@@ -0,0 +1,44 @@
+const ceil = (number, precision) => {
+    let numberString = number.toString();
+    let decimalIndex = numberString.indexOf(".");
+    let beforeDecimal = numberString.substring(0, decimalIndex);
+    let afterDecimal = numberString.substring(
+        decimalIndex,
+        numberString.length
+    );
+    if (Number.isInteger(number)) {
+        return number;
+    }
+    if (precision === undefined || precision === 0) {
```

Comment:
This is another example of when passing in default values to your params can help make your code look cleaner.

consider trying:
```
const ceil = (number, precision=0) => {
```